packages/opentelemetry-sdk-trace-node/test/NodeTracerProvider.test.ts

```diff
@@ -243,4 +245,122 @@ describe('NodeTracerProvider', () => {
       ]);
     });
   });
+
+
+  describe('Custom TracerProvider through inheritance', () => {
```

Comment:
also here could be nice to add test for propagators from `BasicTracerProvider`. It should work to my understanding.

Reply:
I added a similar test to BTP's tests. If it works for BTP, it should work for the NTP.